### PR TITLE
USHIFT-1775: Add router metrics tests

### PR DESCRIPTION
**Pull request by pacevedom**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: origin/test/extended/router/metrics.go [hunk 1]
```diff
@@ -49,6 +49,12 @@ var _ = g.Describe("[sig-network][Feature:Router]", func() {
 	)
 
 	g.BeforeEach(func() {
+		isMicroShift, err := exutil.IsMicroShiftCluster(oc.AdminKubeClient())
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if isMicroShift {
+			g.Skip("MicroShift does not have Prometheus")
```
[the remark this answers is not in the document]
Comment (pacevedom) [reply origin/test/extended/router/metrics.go:55]: These tests require a few more resources than just prometheus. You also need the secrets and service accounts associated with openshift-monitoring stack. Some of the tests might be doable with heavy changes, but I thought it was ok to skip them for now and work our way through.

[the remark this answers is not in the document]
Comment (pmtk) [reply origin/test/extended/router/metrics.go:55]: But we could at least include lines 70-83? That would verify we export metrics port

[the remark this answers is not in the document]
Comment (pacevedom) [reply origin/test/extended/router/metrics.go:55]: That would mean reordering upstream tests too, which I presume difficult, only to skip the test right afterwards. Right now metrics ports are inherited, its not something we officially support, so I would instead wait until we have support for monitoring stack to have the complete tests.

[the remark this answers is not in the document]
Comment (pmtk) [reply origin/test/extended/router/metrics.go:55]: ack

**Pull request merged**